test/run.test.js

```diff
@@ -179,3 +190,22 @@ test('run should callback with after a bailout', (t) => {
     t.end()
   }, 3000)
 })
+
+for (var i = 1; i <= 5; i++) closure(i)
+
+function closure (i) {
+  test('run should count all reply status codes', (t) => {
```

Comment:
change name of test to reflect error code, i.e. 

 ` test(`run should count all ${i}xx status codes`, (t) => {`